frontends/concrete-python/examples/prime-match/README.md

```diff
@@ -0,0 +1,58 @@
+# Inventory Matching System
+
+## Introduction
+
+This tutorial implements two variants of the Inventory Matching System described in this [paper](https://eprint.iacr.org/2023/400.pdf):
+`prime-match.py` implements the classical protocol, while `prime-match-semi-honest.py` follows the semi-honest protocol described in the paper.
```

Comment:
maybe remove `describe in the paper` as it's already written just above

Reply:
done